Deployment/Jenkins/master/Master-Ec2-Instance.groovy

```diff
@@ -0,0 +1,281 @@
+            daysToKeepStr: '30',
+            artifactDaysToKeepStr: '30',
+            artifactNumToKeepStr: '3'
+        )
+    )
+    disableConcurrentBuilds()
+    timeout(time: 60, unit: 'MINUTES')
+}
+
+environment {
+    AWS_DEFAULT_REGION = "${AwsRegion}"
+    AWS_CA_BUNDLE = '/etc/pki/tls/certs/ca-bundle.crt'
+    REQUESTS_CA_BUNDLE = '/etc/pki/tls/certs/ca-bundle.crt'
+}
+
+parameters {
+    string(name: 'AwsRegion', defaultValue: 'us-east-1', description: 'Amazon region to deploy resources into')
+    string(name: 'AwsCred', description: 'Jenkins-stored AWS credential with which to execute cloud-layer commands')
+    string(name: 'GitCred', description: 'Jenkins-stored Git credential with which to execute git commands')
+    string(name: 'GitProjUrl', description: 'SSH URL from which to download the Jenkins git project')
+    string(name: 'GitProjBranch', description: 'Project-branch to use from the Jenkins git project')
+    string(name: 'CfnStackRoot', description: 'Unique token to prepend to all stack-element names')
+    string(name: 'TemplateUrl', description: 'S3-hosted URL for the EC2 template file')
+    string(name: 'ElbName', description: 'ARN of the ELB to attach the EC2 instance')
+    string(name: 'AdminPubkeyURL', description: 'URL the file containing the admin users SSH public keys')
+    string(name: 'AmiDistro', defaultValue: 'CentOS', description: 'Linux distro of the AMI')
+    string(name: 'AmiId', description: 'ID of the AMI to launch')
+    string(name: 'AppVolumeDevice', defaultValue: '', description: 'Device to mount an extra EBS volume. Leave blank to launch without an extra application volume')
+    string(name: 'AppVolumeMountPath', defaultValue: '/var/lib/jenkins', description: 'Filesystem path to mount the extra app volume. Ignored if \"AppVolumeDevice\" is blank')
+    string(name: 'AppVolumeSize', defaultValue: '20', description: 'Size in GB of the EBS volume to create. Ignored if \"AppVolumeDevice\" is blank')
+    string(name: 'AppVolumeType', defaultValue: 'gp2', description: 'Type of EBS volume to create. Ignored if \"AppVolumeDevice\" is blank')
+    string(name: 'BackupBucket', description: 'S3 Bucket to host backups of Jenkins config-data')
+    string(name: 'BackupCronURL', description: 'URL to the Jenkins backup cron-file')
+    string(name: 'BackupFolder', description: 'Folder in S3 Bucket to host backups of Jenkins config-data')
+    string(name: 'CfnEndpointUrl', defaultValue: '', description: 'URL to the CloudFormation Endpoint. e.g. https://cloudformation.us-east-1.amazonaws.com')
+    string(name: 'DnsSuffix', description: 'Suffix for Jenkins hostname and DNS record')
+    string(name: 'EpelRepo', defaultValue: 'epel', description: 'Name of networks EPEL repo')
+    string(name: 'InstanceRole', defaultValue: '', description: 'IAM instance role to apply to the instance')
+    string(name: 'InstanceType', defaultValue: 't2.micro', description: 'Amazon EC2 instance type')
```

Comment:
Probably want to  change this from `.micro` to `.large`

Reply:
changed this to m4.large